File: quadratic.java
import java.util.*;
class Quadratic
{
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter A =");
        int A=sc.nextInt();
        System.out.println("Enter B =");
        int B=sc.nextInt();
        System.out.println("Enter C =");
        int C=sc.nextInt();
        int x1= B*B + 4*A*C;
        int x2= B*B - 4*A*C;
        System.out.println("X1 = "+ x1);
        System.out.println("X2 = "+ x2);

    }
}

File: Quadratic.c
#include<stdio.h>
void main()
{
    int a,b,c;
    float x1,x2;
    printf("Enter the coefficients of quadratic equations: \n");
    scanf("%d%d%d",&a,&b,&c);
    x1=(-b+pow((b*b-4*a*c),0.5))/(2*a);
    x2=(-b-pow((b*b-4*a*c),0.5))/(2*a);
    printf("The roots of the equation are %f and %f.",x1,x2);
}
